app/frontend/components/pages/login/keyFileAuth.js

```diff
@@ -158,67 +158,49 @@ class LoadKeyFileClass extends Component {
         ),
         encrypted &&
           h(
-            'div',
+            Modal,
             {
-              class: 'overlay',
-              onKeyDown: (e) => {
-                e.key === 'Escape' && this.closePasswordModal()
-              },
+              closeHandler: this.closePasswordModal,
```

Comment:
the password modal when logging in by json now does not close by pressing esc button